examples/references/segmentation/pascal_voc2012/code/scripts/training.py

```diff
@@ -1,17 +1,29 @@
 # This a training script launched with py_config_runner
 # It should obligatory contain `run(config, **kwargs)` method
-
+from pathlib import  Path
+import os
 import sys
-from collections.abc import Mapping
-from pathlib import Path
 
+#importing the package parent
+#so no path conflict not happen
+
+PACKAGE_PARENT = '..'
```

Comment:
This should be removed as the same is done below line 37

Reply:
removed it as well.